verifier.sh

```diff
@@ -600,18 +603,24 @@ cd oq-platform-taxtweb
 sudo pip install . -U --no-deps
 cd -
 
+# install taxonomy
+cd oq-platform-taxonomy
+sudo pip install . -U --no-deps
+cd -
+
 echo -e \"y\ny\ny\n\" | oq-platform/openquakeplatform/bin/deploy.sh --hostname oq-platform.localdomain
 
 cd oq-platform/openquakeplatform
 
 # add a simulated qgis uploaded layer
-./openquakeplatform/bin/simqgis-layer-up.sh --sitename "http://oq-platform.localdomain"
+./openquakeplatform/bin/simqgis-layer-up.sh --sitename \"http://oq-platform.localdomain\"
 
 export PYTHONPATH=\$(pwd):\$(pwd)/openquakeplatform/test/config
 sed 's@^pla_basepath *= *\"http://localhost:8000\"@pla_basepath = \"http://oq-platform.localdomain\"@g' openquakeplatform/test/config/moon_config.py.tmpl > openquakeplatform/test/config/moon_config.py
 export DISPLAY=:1
 python -m openquake.moon.nose_runner --failurecatcher prod -v --with-xunit --xunit-file=xunit-platform-prod.xml  openquakeplatform/test
 sleep 3
+sleep 40000 || true
```

Comment:
why `sleep` 40 secs? Isn't too much? And why is in `|| true` it should always return true